src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -166,13 +166,18 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
     firstScheduleDate,
   ]);
 
+  const relatedVenueRooms =
+    relatedVenues.flatMap((venue) => venue.rooms || []).filter((x) => !!x) ||
+    [];
```

Comment:
can you create consts for empty arrays? to have the only created object and not to cause rerenders and recalculation of the dependant objects.

Reply:
> can you create consts for empty arrays? to have the only created object and not to cause rerenders and recalculation of the dependant objects.

There is a constant reference to an object and there is a constant object:
1. `const constantReference = {}`
2. `const constantObject = Object.freeze( {} )`

Passing the first one to subsequent renders and possible those components passing around and modifying will make the example 1 basically be a shared mutable state (quite contrary to React principles)

Can that be achieved with `Object.freeze()` or  `useMemo()` instead?

Reply:
fixed emptyArrayObj